etc/rollup/rollup-plugin-require-vendor/require_vendor.mjs

```diff
@@ -1,9 +1,12 @@
 import MagicString from 'magic-string';
 
-const REQUIRE_POLYFILLS =
-  `const { TextEncoder, TextDecoder } = require('../vendor/text-encoding');
+const REQUIRE_WEB_UTILS_POLYFILLS =
+  `const { TextEncoder } = require('../vendor/text-encoding');
 const { encode: btoa, decode: atob } = require('../vendor/base64');\n`
 
+const REQUIRE_VALIDATE_UTF8_POLYFILLS = 
+  `const { TextDecoder } = require('../vendor/text-encoding');`;
```

Comment:
```suggestion
  `const { TextDecoder } = require('../vendor/text-encoding');\n`;
```
Not a functional change but without this the code produced looks a little strange.

Reply:
I see, made the change!